content/docs/guides/app_onboarding/prereqs.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: "Application Requirements"
+title: "Prerequisites"
```

Comment:
Would be good to reference this in the Quickstart doc as well.

Reply:
Go ahead and add links where you think they belong.